fastlane/Fastfile

```diff
@@ -198,7 +204,7 @@ platform :ios do
     app_store_connect_api_key(
       key_id: DeliverableConstants.APP_STORE_KEY_ID,
       issuer_id: DeliverableConstants.APP_STORE_ISSUER_ID,
-      key_content: Environments.APPSTORE_CONNECT_API_KEY || ''
+      key_filepath: "appstorekey.p8"
```

Comment:
Can you give more insight about using the .p8 file instead of `APPSTORE_CONNECT_API_KEY`?

Reply:
@phongvhd93   Regarding Apple's document about security, we shouldn't store keys in our repository so I think @blyscuit using `p8` file is reasonable

<img width="600" alt="Screen Shot 2022-08-29 at 09 01 48" src="https://user-images.githubusercontent.com/24598204/187108522-6d864c48-f0af-4776-8c19-63362dc7a5c4.png">

Reply:
@phongvhd93  Before we use `APPSTORE_CONNECT_API_KEY` from CI secret, which is already very safe.
Currently fastlane has this bug https://github.com/fastlane/fastlane/issues/20553 that cannot read `APPSTORE_CONNECT_API_KEY` as environment secret.
In this PR we save `APPSTORE_CONNECT_API_KEY` from secret to `.p8` only on the CI machine. This is safe for GH Actions Runner but not safe for self-hosted machine, so we add an extra step to remove the file when the job is done. Value from this .p8 cannot be print, see in POW.
When this bug is fixed by fastlane we can remove these p8 related steps.

Reply:
We NEVER push p8 or APPSTORE_CONNECT_API_KEY directly on the repo.